src/pages/home/report/EmojiPickerMenu/index.native.js

```diff
@@ -0,0 +1,119 @@
+import React, {Component} from 'react';
+import {View, FlatList, Text} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import CONST from '../../../../CONST';
+import styles from '../../../../styles/styles';
+import emojis from '../../../../../assets/emojis';
+import EmojiPickerMenuItem from '../EmojiPickerMenuItem';
+
+const propTypes = {
+    // Function to add the selected emoji to the main compose text input
+    onEmojiSelected: PropTypes.func.isRequired,
+};
+
+class EmojiPickerMenu extends Component {
+    constructor(props) {
+        super(props);
+
+        // Ref for the emoji search input
+        this.searchInput = undefined;
+
+        // This is the number of columns in each row of the picker.
+        // Because of how flatList implements these rows, each row is an index rather than each element
+        // For this reason to make headers work, we need to have the header be the only rendered element in its row
+        // If this number is changed, emojis.js will need to be updated to have the proper number of spacer elements
+        // around each header.
+        this.numColumns = 8;
+
+        // This is the indices of each category of emojis
+        // The positions are static, and are calculated as index/numColumns (8 in our case)
+        // This is because each row of 8 emojis counts as one index
+        // If this emojis are ever added to emojis.js this will need to be updated or things will break
+        this.unfilteredHeaderIndices = [0, 34, 60, 88, 99, 121, 148];
```

Comment:
Can we DRY up `unfilteredHeaderIndices` by putting them in their own file to be imported by `EmojiPickerMenu/index.js` and `EmojiPickerMenu/index.native.js`?